scalableminds/webknossos
# Reactify user list

I reactified the user list using the Antd [table element](https://ant.design/components/table/#rowSelection).  

### Mailable description of changes
- Reworked Admin->User list to include enhanced sorting, filtering and search features
- Defaulted user list to only show activated users

### Steps to test:
1. Sign up a few new users
2. Go to Admin->Users
3. Activate / Deactivate Users
4. Assign Experiences and Teams / Roles

Please check code for immutability vs. direct manipulation gotchas.

### Issues:
- fixes #1916
- fixes #1298 
- Fixes https://discuss.webknossos.org/t/inactive-users-1545/54/3

------
